### DISCUSSION TITLE
Joel Riphagen - Mapping Childhood Vaccination in Sacramento

### DISCUSSION THREAD

**Discussion by giscus** [joelrip/joelrip.github.io#10] [General]
# Joel Riphagen - Mapping Childhood Vaccination in Sacramento

An interactive map of vaccination rates at childcare facilities and Kindergartens in the Sacramento area.

http://localhost:3083/posts/2014-04-20-childhood-vaccination/

**Comment by joelrip** [comment]
_Editor's Note April 2023:_ Giscus comments now added to all blog posts!
